.github/workflows/build.yml

```diff
@@ -89,8 +89,8 @@ jobs:
         platform=${{ matrix.os }}
         platform=${platform/macos-13/macos-latest}
         platform=${platform/macos-14/macos-13-arm64}
+        platform=${platform/windows-2019/windows-latest}
```

Comment:
We should name the new one `windows-arm64`, not `windows-11-arm64` see https://github.com/ruby/setup-ruby/pull/750#discussion_r2059091375